package.json

```diff
@@ -99,6 +99,7 @@
     "medici": "^6.1.0",
     "mongoose": "^6.9.2",
     "node-cache": "^5.1.2",
+    "node-jose": "^2.2.0",
```

Comment:
it should just be in devDependencies?

Reply:
i am using it here https://github.com/GaloyMoney/galoy/pull/2071/files#diff-8c1f181fc0b0efef5b124341ef62eed90f9dcf7f640ecaf7420996ce4d281439R6

Reply:
I only see it used in `dev/ory/gen-test-jwt.ts`